### ISSUE TITLE
DataToolbar toggle groups are not applying aria-haspopup when appropriate

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#2900]
the toggle group button should have aria-haspopup when isExpanded = true && viewport is smaller than lg global breakpoint

**Comment by tlabaj** [comment]
closed by #3012
